.gitignore

```diff
@@ -7,3 +7,7 @@ vendor
 /docker-compose-scripts/
 
 coverage/*
+
+.vscode/
```

Comment:
Not sure if this should be ignored. Any general configuration to vs code dev enviroment should stay the same for every project.